portfolio/src/main/webapp/script.js

```diff
@@ -140,7 +140,13 @@ const mapStyle = [
 /**
  * Calls the methods that load the home page.
  */
-function start() {
+async function start() {
+  const response = await fetch('/login');
+  const loggedIn = await response.json();
+  if (!loggedIn) {
```

Comment:
It seems like we call `getCommentSection()`below regardless of whether this boolean is true or not, but the change to `innerText` in line 147 seems to imply users have to log in to see the section. Is the app behaving as intended? If so, we may just want to add some comments to explain how it all works.

Reply:
The app was actually working as intended on the user's end, but I hadn't noticed this extra call to <addr>getCommentSection()</addr>. Thanks for pointing it out. I fixed this and added a comment to make it clearer.